docs/dev-client-networking.md

```diff
@@ -94,7 +94,7 @@ Hubs Client manages who the creator of `networked instanciated entities`,
 `networked entities` instanciated by it are removed from the room.
 
 In order to keep these entities stayed in the room even after their authors
-leave a room, the entity must be [`pinned`](https://hubs.mozilla.com/docs/hubs-features.html).
+leave a room, the entity must be pinned.

```

Comment:
I think this could just be changed to this?
```suggestion
leave a room, the entity must be [`pinned`](https://docs.hubsfoundation.org/hubs-features.html).

```